tests/golden/copy-command/copy-command.bats

```diff
@@ -174,3 +174,56 @@ EOF
       y - [2000-01-01 01:01:01]
 EOF
 }
+
+@test "copying a directory fails when any entry clashes with an existing directory" {
+  coffer create /a/b
+  coffer create /x/b/c
+
+  run coffer copy /a /x
+
+  assert_failure
+  assert_output - <<EOF
+[ERROR] The following entries cannot be copied because a directory already exists at the destination.
+
+Cannot copy '/a/b' to '/x/b'.
+EOF
+
+  run coffer copy /a /x -f
+
+  assert_failure
+  assert_output - <<EOF
+[ERROR] The following entries cannot be copied because a directory already exists at the destination.
+
+Cannot copy '/a/b' to '/x/b'.
+EOF
```

Comment:
Since we're executing two commands in quick succession and we expect them both to return the exact same output, I think it makes sense to use a variable to store the expected output:

```suggestion
  EXPECTED=$(cat <<EOF
[ERROR] The following entries cannot be copied because a directory already exists at the destination.

Cannot copy '/a/b' to '/x/b'.
EOF
)
  run coffer copy /a /x
  assert_failure
  assert_output "$EXPECTED"

  run coffer copy /a /x -f
  assert_failure
  assert_output "$EXPECTED"
```

Reply:
Done